src/test/client/test_region_meta.py

```diff
@@ -41,7 +42,7 @@ def test_region_params_wrap(self, mock_connect):
         region_item = self.region_meta._region_params(REGION)
         self.assertEqual(region_item['az1'], 'us-east-1a')
         self.assertEqual(region_item['az2'], 'us-east-1c')
-        self.assertEqual(region_item['az3'], 'us-east-1a')
+        self.assertNotIn('az3', region_item)
 
```

Comment:
The idea was to always resolve 3 AZs even if there were only 2 available in the region; but I think this would hit the same `ELB cannot be attached to multiple subnets in the same AZ` issue that you did.

I blame me for usually testing in us-east-1 ($$$)